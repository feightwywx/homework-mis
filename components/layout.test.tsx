import { render } from '@testing-library/react'
import HwLayout from './layout'

describe('Layout', () => {
  it('renders HwLayout', () => {
    const { container } = render(<HwLayout>
      <div>test</div>
    </HwLayout>);
    expect(container).toMatchSnapshot();
  })
})