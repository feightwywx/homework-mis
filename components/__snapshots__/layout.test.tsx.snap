// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders HwLayout 1`] = `
<div>
  <section
    class="ant-layout"
  >
    <header
      class="ant-layout-header"
    >
      <div
        class="logo"
      >
        学生作业管理系统
      </div>
    </header>
    <main
      class="ant-layout-content layout_content"
    >
      <div
        class="ant-spin-nested-loading"
      >
        <div
          class="ant-spin-container"
        >
          <div
            class="site-layout-content"
          >
            <div>
              test
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="ant-layout-footer"
      style="text-align: center;"
    >
      ©️2022 .direwolf
    </footer>
  </section>
</div>
`;
